src/libraries/System.Private.CoreLib/src/Microsoft/Win32/SafeHandles/SafeFileHandle.Unix.cs

```diff
@@ -64,26 +64,31 @@ private static SafeFileHandle Open(string path, Interop.Sys.OpenFlags flags, int
 
             // Make sure it's not a directory; we do this after opening it once we have a file descriptor
             // to avoid race conditions.
-            Interop.Sys.FileStatus status;
-            if (Interop.Sys.FStat(handle, out status) != 0)
+            // 
```

Comment:
"Code should not contain trailing whitespace"

```suggestion
            //
```